components/raftstore-v2/src/raft/peer.rs

```diff
@@ -85,7 +85,11 @@ pub struct Peer<EK: KvEngine, ER: RaftEngine> {
 
     /// region buckets.
     region_buckets: Option<BucketStat>,
+    // the last buckets records the stats that the recently refreshed.
     last_region_buckets: Option<BucketStat>,
+    // the report region buckets records the increment stats after last report pd.
+    // it will be reset after report pd.
+    report_region_buckets: Option<BucketStat>,
```

Comment:
I think these three fields can be extracted to a struct and defined in bucket.rs instead.

Reply:
good idea, I will fix it .